src/uu/df/src/blocks.rs

```diff
@@ -245,6 +245,7 @@ mod tests {
         assert_eq!(to_magnitude_and_suffix(1001).unwrap(), "1.1kB");
         assert_eq!(to_magnitude_and_suffix(1023).unwrap(), "1.1kB");
         assert_eq!(to_magnitude_and_suffix(1025).unwrap(), "1.1kB");
+        assert_eq!(to_magnitude_and_suffix(10_001).unwrap(), "11kB");
```

Comment:
So in this case, the quotient of `10_001` and `1_000` is `10.001`, which gets rounded up to `11` due to the new condition added to the code. Interesting.